### PR TITLE
feature: Added locked deposit

### PR DESCRIPTION
**Pull request by stackchain**

### CODE REVIEW

File: src/TxHistory/LockedDeposit.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  const intl = useIntl()
+
+  return {
+    lockedDeposit: intl.formatMessage(globalMessages.lockedDeposit),
+  }
+}
+
+/**
+ * Calculate the lovelace locked up to hold utxos with assets
+ * Important `minAdaRequired` is missing `has_hash_data`
```
Comment (stackchain) [inline src/TxHistory/LockedDeposit.tsx:81]: Needs serialization lib v10

File: src/TxHistory/LockedDeposit.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      }}
+    >
+      <FormattedLockedAmount amount={amount} />
+    </Boundary>
+  )
+}
+
+function FormattedLockedAmount({amount}: {amount: string}) {
+  return (
+    <AlignHorizontal>
```
Comment (wolverineks) [inline src/TxHistory/LockedDeposit.tsx]: AlignHorizonal -> Row

File: src/TxHistory/LockedDeposit.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  const query = useQuery({
+    suspense: true,
+    queryKey: [wallet.id, 'lockedAmount'],
+    ...options,
+    queryFn: async () => {
+      const networkConfig = getCardanoNetworkConfigById(wallet.networkId)
+      const minUtxoValue = await BigNum.fromStr(networkConfig.MINIMUM_UTXO_VAL)
+      const utxos = await wallet.fetchUTXOs()
+      const utxosWithAssets = utxos.filter((u) => u.assets.length > 0)
+      const promises = utxosWithAssets.map(async (u) => {
```
Comment (wolverineks) [inline src/TxHistory/LockedDeposit.tsx]: can this be abstracted somewhere else?

File: src/legacy/utils.ts [hunk 4]
```diff
@@ -333,7 +333,8 @@ export const cardanoValueFromRemoteFormat = async (utxo: RawUtxo) => {
 
   for (const entry of utxo.assets) {
     const {policyId, name} = await identifierToCardanoAsset(entry.assetId)
-    const policyContent = (await assets.get(policyId)) ?? (await Assets.new())
+    let policyContent = await assets.get(policyId)
```
Comment (wolverineks) [inline src/legacy/utils.ts:336]: why was this necessary?

File: src/yoroi-wallets/cardano/types.ts [hunk 5]
```diff
@@ -302,6 +302,7 @@ type YoroiWalletKeys =
   | 'publicKeyHex'
   | 'subscribe'
   | 'toJSON'
+  | 'asAddressedUtxo'
```
Comment (wolverineks) [inline src/yoroi-wallets/cardano/types.ts]: YoroiWallet should approach blockchain agnostic

**Review by wolverineks — CHANGES_REQUESTED** [review]

File: src/legacy/utils.ts (hunk 4 above)
Comment (stackchain) [reply src/legacy/utils.ts:336]: Since one asset is a combination of `policyId` + `name`, it maps to build the `Value` object that minAda expects.

**Review by banklesss — APPROVED** [review]

**Pull request merged**